client/my-sites/plans/plans.jsx

```diff
@@ -0,0 +1,28 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import config from 'config';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import getIntervalTypeForTerm from 'state/plans/get-interval-type-for-term';
+import getCurrentPlanTerm from 'state/selectors/get-current-plan-term';
+import Plans from 'my-sites/plans/main';
+
+export default connect( ( state, { intervalType } ) => {
```

Comment:
We usually keep the `connect()` calls in the same file we define the component in - mostly because the unconnected component is usually included only in tests.

Reply:
This is a `connect()` over a already `connect()`-ed component so we can have a different `<Plans />` component only for purposes of wp.com and leave jetpack with the regular one. It was discussed with @seear and seems like the best solution in this case. The pattern of defining it in a separate file was borrowed from jetpack where a similar things exists :-)